common/supplemental/likelySubtags.xml

```diff
@@ -1928,8 +1950,6 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<!--{ Nzima; ?; ? } => { Nzima; Latin; Ghana }-->
 		<likelySubtag from="oc" to="oc_Latn_FR"/>
 		<!--{ Occitan; ?; ? } => { Occitan; Latin; France }-->
-		<likelySubtag from="oc_ES" to="oc_Latn_ES"/>
-		<!--{ Occitan; ? ; Spain } => { Occitan; Latin; Spain }-->
```

Comment:
I think this removal is OK because the script is already picked up from the previous line… correct?

Reply:
Yes. Changes like this should be the result of a population increase that tips the scales to ES. So you need to check that that is the case.

Reply:
This is already ES

Reply:
this is already in _ES so no scale tipping needed.   Again this was a manual addition (from years back) and I think it's just not needed.